tests/TestUtils.inc.php

```diff
@@ -16,8 +16,9 @@ static function LoadJSONSchema($filename) {
     }
 
     static function Hostname() {
-      if(isset($_SERVER['api_keyman_com_host'])) {
-        return $_SERVER['api_keyman_com_host'];
+      $env = getenv();
+      if(isset($env['api_keyman_com_host'])) {
+        return $env['api_keyman_com_host'];
```

Comment:
I don't know what the cost of `getenv()` is but this could potentially be slow if we call `Hostname()` a lot?